app/src/main/res/values/strings.xml

```diff
@@ -198,6 +198,11 @@
     <string name="download_file_not_supported">Download protocol not supported</string>
     <string name="cannot_find_the_file">Cannot find the file</string>
     <string name="cannot_delete_the_file">Cannot delete the file.</string>
+    <string name="download_completed">\u0020downloaded</string>
```

Comment:
Use %1$s to form a complete phrase, since "downloaded" may come in front of the file name for some locales.